packages/config/src/config/build.js

```diff
@@ -16,10 +16,10 @@ export default () => ({
     // { isDev, isClient, isServer }
     app: ({ isDev, isModern }) => isDev ? `[name]${isModern ? '.modern' : ''}.js` : `[contenthash:7]${isModern ? '.modern' : ''}.js`,
     chunk: ({ isDev, isModern }) => isDev ? `[name]${isModern ? '.modern' : ''}.js` : `[contenthash:7]${isModern ? '.modern' : ''}.js`,
-    css: ({ isDev }) => isDev ? '[name].css' : '[name].[contenthash:7].css',
-    img: ({ isDev }) => isDev ? '[path][name].[ext]' : 'img/[name].[contenthash:7].[ext]',
```

Comment:
We had this from before due to SEO friendliness. I think removing only from css to resolve module naming issue should be enough :)